lib/utils/constants.dart

```diff
@@ -0,0 +1,9 @@
+class Constants {
```

Comment:
Jednak nazwij to assets i zostaw assety i zrób nowy plik o nazwie Dimens i tam przechowuj wartości. I wszystkie wartości jakie masz w plikach (width i height) daj do tego pliku o nazwach space_liczba_